### DISCUSSION TITLE
DatePicker Height

### DISCUSSION THREAD

**Discussion by superpandaaa** [rsuite/rsuite#4035] [Q&A]
Hi team,

I am new to rsuite and trying to apply DatePicker to my React project. And I am able to adjust the width of the DatePicker component, I am wondering it that possible to adjust the height as well. I have try the following but it only applys to the wrapper, not the input field inside it so it will be shown as original height. 

Much appreciated

```
<DatePicker format="MM/dd/yyyy" style={{width: "300px", height: "39px"}}/>

// and in css i try to override it like below
.rs-input-group .rs-input-group-inside {
    height: 39px !important;
}

```

**Comment by ehcaneontheweb** [comment]
Your CSS does not seem to work as expected. I have it a try in a browser and you only need something like this. 

.rs-input-group  {
    height: 39px ;
}
